src/Common/test-runtime/XUnit.Runtime.depproj

```diff
@@ -23,7 +23,12 @@
                                                                               '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='xunit.extensibility.execution' And 
                                                                               '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='xunit.console.netcore' And 
                                                                               '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='xunit.runner.utility' And 
-                                                                              '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='Microsoft.xunit.netcore.extensions'" />
+                                                                              '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='Microsoft.xunit.netcore.extensions' And
+                                                                              '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='runtime.win7-x64.Microsoft.NETCore.Runtime.CoreCLR' And
```

Comment:
Same here, I think this is better to have as a local stash, but I don't believe we should check this in. cc @weshaggard